e2e/client/playwright.config.ts

```diff
@@ -13,7 +13,7 @@ import path from 'path';
 export default defineConfig({
     testDir: './playwright',
     /* Run tests in files in parallel */
-    fullyParallel: true,
+    fullyParallel: false,
```

Comment:
why are we no longer gonna do parallel?

Reply:
because that's causing tests to fail when there are 2 tests modifying data